packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -139,13 +139,21 @@ describe('plugin-meeting', () => {
   });
 
   // Alice calls bob and bob rejects it
-  describe('Successful 1:1 meeting (including Guest)', () => {
+  describe('Successful 1:1 meeting (including Guest)', function () {
```

Comment:
Was `function` needed for the scope of `this`?

Reply:
yeah we leave it to debug and add a name